src/STEMMUS_SCOPE.m

```diff
@@ -788,6 +802,7 @@
                                       Evap, WaterStressFactor, WaterPotential, Sim_hh, Sim_qlh, Sim_qlt, Sim_qvh, ...
                                       Sim_qvt, Sim_qla, Sim_qva, Sim_qtot, ForcingData, RS, RWUs, RWUg);
         fclose("all");
+        io.checkModelRun(KT, endTime); % display model running per time step
```

Comment:
```suggestion
```

Reply:
Please avoid adding `disp` messages to the main loop of the model. Doing so will increase the size of the command output sometimes to GB, especially when processing long time series for multiple locations on HPC, and can cause issues when running the model in parallel. If you need to capture this information, it should be logged to a separate file with a specific filename. or you can write a test script under directory `tests`.

Reply:
> If you need to capture this information, it should be logged to a separate file with a specific filename. or you can write a test script under directory `tests`.

Could you please guide me how to log this information to a separate file with a specific filename? I indeed need this info, especially in long time-series models to have a track of the running time